config/v1/types_scheduling.go

```diff
@@ -48,6 +48,14 @@ type SchedulerSpec struct {
 	// would not be applied.
 	// +optional
 	DefaultNodeSelector string `json:"defaultNodeSelector,omitempty"`
+	// MastersSchedulable allows masters nodes to be schedulable. This means, if there
+	// are n no of master nodes in the cluster, all of them would be made schedulable.
```

Comment:
why mention n? "This means that all of them would be made schedulable."

Reply:
Changed it to 10 as suggested by @deads2k